avaframe/out3Plot/outAIMEC.py

```diff
@@ -97,33 +96,94 @@ def visuTransfo(rasterTransfo, inputData, cfgPath, cfgFlags):
     ax2.legend(loc=4)
     pU.addColorBar(im, ax2, ticks, pU.cfgPlotUtils['unitppr'])
 
-    outFileName = '_'.join([projectName, dirName, 'DomainTransformation'])
+    #outFileName = '_'.join([projectName, dirName, 'DomainTransformation'])
+    outFileName = '_'.join([projectName, 'DomainTransformation'])
     pU.saveAndOrPlot(cfgPath, cfgFlags, outFileName, fig)
 
 
-def visuRunout(rasterTransfo, resAnalysis, plim, newRasters, cfgPath, cfgFlags):
+def visuRunoutComp(rasterTransfo, resAnalysis, plim, newRasters, cfgPath, cfgFlags):
     """
     Plot and save the Peak Pressure  distribution after coord transfo
     """
     ####################################
     # Get input data
     # read paths
     projectName = cfgPath['projectName']
-    dirName = cfgPath['dirName']
+    #dirName = cfgPath['dirName']
+    # read data
+    s = rasterTransfo['s']
+    l = rasterTransfo['l']
+    PPRCrossMax = resAnalysis['PPRCrossMax']
+    PPRCrossMean = resAnalysis['PPRCrossMean']
+    PFDCrossMax = resAnalysis['PFDCrossMax']
+    PFDCrossMean = resAnalysis['PFDCrossMean']
+    PFVCrossMax = resAnalysis['PFVCrossMax']
+    PFVCrossMean = resAnalysis['PFVCrossMean']
+
+    ############################################
+    # prepare for plot
+    Title = ['Pressure ', 'Flow Depth ', 'Flow Velocity ']
+    Unit = ['$P(s)$ [kPa]', '$fd(s)$ [m]', '$v(s) [m.s^{-1}]$']
+    DataMax = np.array(([None] * 3))
+    DataMax[0] = PPRCrossMax
+    DataMax[1] = PFDCrossMax
+    DataMax[2] = PFVCrossMax
+
+    DataMean = np.array(([None] * 3))
+    DataMean[0] = PPRCrossMean
+    DataMean[1] = PFDCrossMean
+    DataMean[2] = PFVCrossMean
+
+    ############################################
+    # Figure: Pressure depth speed
+
+    fig, axes = plt.subplots(nrows=1, ncols=3, figsize=(pU.figW*3, pU.figH))
+    fig.subplots_adjust(left=0.05, bottom=0.05, right=0.95, top=0.95, hspace=0.3)
+
+    for ax, dataMax, dataMean, title, unit in zip(axes.flatten(), DataMax, DataMean, Title, Unit):
+        ax.plot(dataMax[0, :], s, '--k', label='Max Reference')
+        ax.plot(dataMean[0, :], s, '-k', label='Mean Reference')
+        ax.plot(dataMax[1, :], s, '--b', label='Max Simulation')
+        ax.plot(dataMean[1, :], s, '-b', label='Mean Simulation')
+
+        ax.set_title(title + 'distribution along the path')
```

Comment:
Please adjust variable naming: `DataMax` to `dataMax` etc...

Reply:
moved to issue